malware-family/donut-loader/load-shellcode-via-donut.yml

```diff
@@ -0,0 +1,62 @@
+rule:
+  meta:
+    name: load shellcode via donut
+    namespace: malware-family/donut-loader
+    authors:
+      - still@teamt5.org
+    scopes:
+      static: file
+      dynamic: unsupported  # requires mnemonic features
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+      - Defense Evasion::Obfuscated Files or Information::Software Packing [T1027.002]
+      - Execution::Native API [T1106]
+      - Execution::Command and Scripting Interpreter::Visual Basic [T1059.005]
+      - Execution::Command and Scripting Interpreter::JavaScript [T1059.007]
+      - Execution::Inter-Process Communication::Component Object Model [T1559.001]
+    examples:
+      - d890c1c67d83f1131c065b5eb5f263cbf54559dbcdb4562c3bde3dc30d1a3205
+  features:
+    - 3 or more:
+      - match: encrypt data using chaskey
+      - match: encrypt data using speck
+      - match: load assembly via IAssembly
+      - match: decompress data using aPLib
+      - function:
```

Comment:
Can this (or a subset of features) be tightened with _basic block_ to avoid potential FPs?